generator/overrides/education/pages/issuer/credential-form/index.tsx

```diff
@@ -16,7 +17,11 @@ const CredentialForm: FC = () => {
 
   useEffect(() => {
     if (error) {
-      showErrorToast(new Error(messages.issuer.error.apiError))
+      if (error.response?.data?.error?.code === ErrorCodes.INTERNAL_SERVER_ERROR) {
+        showErrorToast(new Error(messages.issuer.error.apiError))
+        return
+      }
+      showErrorToast(error)
```

Comment:
```suggestion
      if (error.response?.data?.error?.code === ErrorCodes.INTERNAL_SERVER_ERROR) {
        showErrorToast(new Error(messages.issuer.error.apiError))
      } else {
        showErrorToast(error)
      }
```

Reply:
and all other places with the same coditions

Reply:
Usually, we don't need to explicitly check that code equals INTERNAL_SERVER_ERROR. Usually this is reserved for cases when we couldn't handle the error:
```ts
if (error.code === 'ERROR_1') {

} else if (error.code === 'ERROR_2') {

} else {
  // assume it's internal server error or any unhandled error
}
```

I'm not sure what's the difference here between `showErrorToast(new Error(messages.issuer.error.apiError))` and `showErrorToast(error)`. Can't we just do this here?
```ts
if (error) {
  showErrorToast(new Error(messages.issuer.error.apiError))
}
```